connect/runtime/src/main/java/org/apache/kafka/connect/runtime/TopicCreationConfig.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.connect.runtime;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigException;
+import org.apache.kafka.connect.util.TopicAdmin;
+
+import java.util.Collections;
+
+public class TopicCreationConfig {
+
+    public static final String DEFAULT_TOPIC_CREATION_PREFIX = "topic.creation.default.";
+    public static final String DEFAULT_TOPIC_CREATION_GROUP = "default";
+
+    public static final String INCLUDE_REGEX_CONFIG = "include";
+    private static final String INCLUDE_REGEX_DOC = "A list of strings that represent regular "
+            + "expressions that may match topic names. This list is used to include topics that "
+            + "match their values and apply this group's specific configuration to the topics "
+            + "that match this inclusion list. $alias applies to any group defined in topic"
+            + ".creation.groups but not the default";
```

Comment:
I know you're following the text that was in the KIP, which made sense because `$alias` appeared in the property key rule in the table. However, as a user the appearance of `$alias` here would be really hard to follow. I'd argue since we're dynamically adding configkeys for all topic creation group aliases and therefore the doc will apply to a specific include key (e.g., `topic.creation.rule1.include`), we should remove this sentence from the documentation of each of the generated configkeys.
```suggestion
            + "that match this inclusion list.";
```

Same with the other documentation constants in this class.

Reply:
Good point. We could always substitute with `String.format` but since it doesn't add much, I've removed it here and below.